### ISSUE TITLE
Problem with loading models

### ISSUE THREAD

**Issue by amizadeh** [Microsoft/CNTK#2317]
Hi,

I'm using CNTK 2.1 CPU-only version on Windows 10. I've loaded the pre-trained ResNet50 (on ImageNet) using the Python API and saved it back to a **new file** using Python API again. I can load the new file (theoretically the same model) using Python, but when I try to do so using BrainScript, I get the following error:

Load: Loading model file: Models/r50.model

[CALL STACK]
    > 00007FF7287A2E84 (SymFromAddr() error: Attempt to access invalid address.)
    - 00007FF72879FAAD (SymFromAddr() error: Attempt to access invalid address.)
    - 00007FF7286E5E63 (SymFromAddr() error: Attempt to access invalid address.)
    - 00007FF7286EA393 (SymFromAddr() error: Attempt to access invalid address.)
    - 00007FF7286CED18 (SymFromAddr() error: Attempt to access invalid address.)
    - 00007FF7286CD502 (SymFromAddr() error: Attempt to access invalid address.)
    - 00007FF7286CC325 (SymFromAddr() error: Attempt to access invalid address.)
    - 00007FF7286CB181 (SymFromAddr() error: Attempt to access invalid address.)
    - 00007FF7286D0304 (SymFromAddr() error: Attempt to access invalid address.)
    - 00007FF7286D01B6 (SymFromAddr() error: Attempt to access invalid address.)
    - 00007FF7286D0099 (SymFromAddr() error: Attempt to access invalid address.)
    - 00007FF7286CF8AC (SymFromAddr() error: Attempt to access invalid address.)
    - 00007FF7284F668D (SymFromAddr() error: Attempt to access invalid address.)
    - 00007FF7284EF543 (SymFromAddr() error: Attempt to access invalid address.)
    - 00007FF7284F27B6 (SymFromAddr() error: Attempt to access invalid address.)
    - 00007FF728506C39 (SymFromAddr() error: Attempt to access invalid address.)

EXCEPTION occurred: error reading string or missing 0: No error

Here's my BrianScript code:
`command = Eval

precision = "float"; traceLevel = 1 ; deviceId = "auto"

rootDir = "."
configDir = "$RootDir$"
dataDir = "$RootDir$"
outputDir = "$RootDir$/Output"
modelDir = "$OutputDir$/Models"

meanDir = "C:/CNTK-Samples-2-0-beta12-0/Examples/Image/Classification/ResNet/Python"
modelPath = "$modelDir$/ResNet_50"
stderr = "$outputDir$/ResNet_50_BS_out"

inputModelPath = "Models/r50.model"
#inputModelPath = "Models/ResNet50_ImageNet_CNTK.model"

Eval = {
    action = "test"
    modelPath = "$modelPath$.PBN"
    minibatchSize = 200
    evalNodeNames = errs:top5Errs

	BrainScriptNetworkBuilder = {
        imageShape  = 224:224:3                 # image dimensions
        labelDim    = 1000                      # number of distinct labels

        model = BS.Network.Load ("$inputModelPath$")
		featExt = BS.Network.CloneFunction (model.features, model.z, parameters="learnable")

        # inputs
        features    = Input {imageShape}
        labels      = Input {labelDim}

        # apply model to features
        z           = featExt (features)

        # loss and error computation
        ce          = CrossEntropyWithSoftmax   (labels, z)
        errs        = ClassificationError       (labels, z)
        top5Errs    = ClassificationError       (labels, z, topN = 5)

        # declare special nodes
        featureNodes    = (features)
        labelNodes      = (labels)
        criterionNodes  = (ce)
        evaluationNodes = (errs) # top5Errs only used in Eval
        outputNodes     = (z)
    }

    reader={
        readerType = "ImageReader"
        file = "$DataDir$/train_small_map.txt"
        randomize = "None"
        features = {
            width = 224
            height = 224
            channels = 3
            cropType = "Center"
            sideRatio = 0.875
            meanFile = "$meanDir$/ImageNet1K_mean.xml"
        }
        labels = {
            labelDim = 1000
        }
    } 
}`

Surprisingly, the script doesn't have any issue with loading the original file! What am I missing here?

Thank you in advance!

**Comment by cha-zhang** [comment]
Python API will save the model to v2 format, which is not supported loading with BrainScript. 

You can save models in v1 format but this is meant for debugging purpose, and it will likely be deprecated soon. 
https://www.cntk.ai/pythondocs/cntk.debugging.debug.html?highlight=legacy#cntk.debugging.debug.save_as_legacy_model

**Comment by amizadeh** [comment]
Thank you very much! That resolved the issue. 
The only reason I'm using BrainScript is to fine-tune a ResNet on the ImageNet dataset using the BrainScript code provided on this repo. I can try to create the Python equivalent of the provided script but of course, it'd be much easier if such Python code already exists somewhere on the repo that I'm not aware of.

**Comment by n17s** [comment]
Closing. Feel free to reopen.

**Comment by Brandonnogithub** [comment]
why it is still v2 format
